docs/source/architecture/character.rst

```diff
@@ -0,0 +1,100 @@
+Character Concepts
+==================
+
+NuCypher's API is written as characters within a cryptological narrative/archetype, each fulfilling a particular
+role.
+
+* **Alice**:  *"The Data Owner"* - retains full control over the data encrypted for her and determines whom to share the data with
+* **Enrico**: *"The Encryptor"* - a data source that encrypts data on behalf of Alice
+* **Bob**:    *"The Data Recipient"* - the data recipient that Alice intends to share data with
+* **Ursula**: *"The Proxy in PRE"* - the nodes on the NuCypher Network that stand ready to re-encrypt data in exchange for payment in fees and token rewards; they enforce the access policy created by Alice
+
+.. _Umbral: https://github.com/nucypher/umbral-doc/blob/master/umbral-doc.pdf
+
+
+Character Actions
+-----------------
+
+
+Alice::Grant
+^^^^^^^^^^^^
+
+.. image:: ../.static/img/alice_grant.svg
+    :target: ../.static/img/alice_grant.svg
+
+Alice may have data that she wants to share and it is conceivable that she would want to classify/categorize
+her data according to how she intends to share it. Each of these subsets should have their own set of permissions - within NuCypher
```

Comment:
I think this opening paragraph reads with a tone of uncertainty.   "Alice has data she needs to share and label."

Reply:
✔️